release-notes.md

```diff
@@ -69,14 +69,16 @@ The following updates were made in this release:
 * Added support for attach data rules consequence.
 * Added support for a boolean-type comparison for the `Exist` or `Not Exist` rules condition.
 
-**Android Core 1.4.5**
+**Android Core 1.4.5 (sdk-core:1.4.6)**
 
 * Added support for attach data rules consequence.
 * Added support for a boolean-type comparison for the `Exist` or `Not Exist` rules condition.
 * Fixed a bug where the `Exist` and `Not Exist` rules condition might not work for a `List` or `Map` type value.
 * Fixed a bug that, when fetching the remote config from Launch, might cause a crash on some Android devices.
 * Fixed a bug that, when the data URL of an activity did not contain valid schema, might cause a crash.
 
+These changes were released as part of `sdk-core` v1.4.6.
```

Comment:
- please replace "(sdk-core:1.4.6)" with (sdk-core version 1.4.6)"

- Do we need the final sentence ("These changes were released as part of `sdk-core` v1.4.6.")? We already state that it's an Android core version 1.4..6 at the beginning.